src/Front/Parser.y

```diff
@@ -314,8 +314,18 @@ exprAtom
   | id                                  { Id $1 }
   | '(' expr ')'                        { $2 }
   | '(' ')'                             { Lit LitEvent }
+  | '[' exprList ']'                    { ListExpr $2 }
   | '(' expr ',' exprTups ')'           { Tuple ($2 : $4)  }
 
+-- | List Expression.
+exprList
+  :list                                           { $1 }
+  | {- empty list -}                               { [] }
+
+list
+  :exprAtom                                 { [$1] }
+  | exprAtom ',' list                       { $1 : $3 }
```

Comment:
I can't remember if we ever got the formatter working on your vs code. Can you run the formatter over this file? Or at least make the curly braces on lines 322, 323, 326, and 327 align with the curly braces starting on line 318 and above?